src/Storage/Contracts/Criteria.php

```diff
@@ -0,0 +1,14 @@
+<?php
+
+namespace EthicalJobs\SDK\Storage\Contracts;
```

Comment:
These contracts are copied from laravel-storage, right?

Is our code going to properly interoperate with Laravel if we're using copies of their types, rather than the types from the original package? 🤔

Reply:
I had assumed laravel-storage was a Laravel component, but it's actually a package of our own devising, and there's no real reason to have both this new version of sdk-php *and* laravel-storage installed in the same project. So compatibility between the two versions isn't a concern.

Cool.